src/main/java/ogc/rs/apiserver/ApiServerVerticle.java

```diff
@@ -1206,7 +1202,150 @@ public void getStacCollection(RoutingContext routingContext) {
                 routingContext.put("statusCode", 500);
                 routingContext.next();
               }
-              routingContext.put("response", jsonResult.toString());
+              routingContext.put("response", collection.toString());
+              routingContext.put("statusCode", 200);
+              routingContext.next();
+            })
+        .onFailure(
+            failed -> {
+              if (failed instanceof OgcException) {
+                routingContext.put("response", ((OgcException) failed).getJson().toString());
+                routingContext.put("statusCode", ((OgcException) failed).getStatusCode());
+              } else {
+                OgcException ogcException =
+                    new OgcException(500, "Internal Server Error", "Internal Server Error");
+                routingContext.put("response", ogcException.getJson().toString());
+                routingContext.put("statusCode", ogcException.getStatusCode());
+              }
+              routingContext.next();
+            });
+  }
+
+  // /items for stac_collection
+  public void getStacItems(RoutingContext routingContext){
+//    String stacCollectionId = routingContext.normalizedPath().split("/")[3];
+    String stacCollectionId = routingContext.pathParam("collectionId");
+    LOGGER.debug("collectionId- {}", stacCollectionId);
+    JsonObject featureCollections = new JsonObject();
+    JsonArray commonLinksInFeature = new JsonArray()
+        .add(new JsonObject()
+            .put("rel", "collection")
+            .put("type", "application/json")
+            .put("href", stacMetaJson.getString("hostname") + "/stac/collections/" + stacCollectionId))
+        .add(new JsonObject()
+            .put("rel", "parent")
+            .put("type", "application/json")
+            .put("href", stacMetaJson.getString("hostname") + "/stac/collections/" + stacCollectionId))
+        .add(new JsonObject()
+            .put("rel", "root")
+            .put("type", "application/json")
+            .put("href", stacMetaJson.getString("hostname") + "/stac"));
+    dbService
+        .getStacItems(stacCollectionId)
+        .onSuccess(
+            stacItems -> {
+              LOGGER.debug("Success! - {}", stacItems.toString());
+              try {
+                stacItems.forEach(stacItem -> {
+                  JsonArray allLinksInFeature = new JsonArray()
+                      .add(commonLinksInFeature)
+                      .add(new JsonObject()
+                          .put("rel", "self")
+                          .put("type", "application/json")
+                          .put("href", stacMetaJson.getString("hostname")
+                              + "/stac/collections/" + stacCollectionId + "/items/" + stacItem.getString("id")));
+                  stacItem.put("links", allLinksInFeature);
+                });
+                featureCollections.put("features", stacItems);
+                featureCollections.put("links", new JsonArray()
```

Comment:
Need to modify asset links to create `hostname/assets/<asset-id>` URL like in `stacCollections`